src/pages/Admin/Charity/EditProfile/schema.ts

```diff
@@ -48,4 +48,5 @@ export const schema = object<any, SchemaShape<FV>>({
     tiktok: url,
   }),
   overview: richTextContent({ maxChars: MAX_CHARS }),
+  slug: string().trim().matches(/\D|^$/, "can't use id as slug"),
```

Comment:
Not sure what the backend logic will be, but do we want to allow users to be able to remove their slug once it has been set?
`user has no slug -> user sets some slug like "some-slug" -> user updates profile to remove the slug (sets to empty string)`

Reply:
user setting slug to `""` is simply treated as endow doesn't have slug at all. If no slug is passed on `GET` endow isn't find by `slug`.